tsconfig.json

```diff
@@ -16,7 +16,8 @@
   "inlineSourceMap": false,
   "inlineSources": true,
   "declaration": false,
+  "outDir": "./dist",
   // This avoids building the files.
-  "noEmit": true,
+  // "noEmit": true,
```

Comment:
Maybe worth removing this?

Reply:
I will do some testing, but I think that ts-node doesn't emit, I couldn't see a command line switch for it.
I added `--noEmit` to the `npm run lint` command, so that it doesn't emit.
We need to emit the code for the deployment methods on Heroku.
We _might_ be able to run ts-node on Heroku, but I was just following the patterns I had in place on my other bot.